module/zfs/spa_log_spacemap.c

```diff
@@ -1185,6 +1186,16 @@ spa_ld_log_sm_data(spa_t *spa)
 		summary_add_data(spa, sls->sls_txg,
 		    sls->sls_mscount, 0, sls->sls_nblocks);
 
+		char buf[256];
+		(void) snprintf(buf, sizeof (buf),
+		    "read %llu of %llu log space maps "
+		    "in %lld ms", (u_longlong_t)nsm,
+		    (u_longlong_t)avl_numnodes(&spa->spa_sm_logs_by_txg),
+		    (longlong_t)((gethrtime() - read_logs_starttime)
+		    / 1000000));
+
+		spa_import_progress_set_notes(spa_guid(spa), buf);
```

Comment:
There may be up to 1000 spacemaps. It may get noisy.

Only a cosmetics, but I wonder why you inserted this block here, but neither at beginning, nor after prefetch where we may have time to waste, nor after load about where you increment nsm?

Reply:
yes, good point.  Will change to use spa_import_progress_set_notes_nolog()